sdk/textanalytics/ai-text-analytics/README.md

```diff
@@ -104,404 +57,7 @@ const client = new TextAnalyticsClient("<endpoint>", new DefaultAzureCredential(
 
 ### TextAnalyticsClient
 
-`TextAnalyticsClient` is the primary interface for developers using the Text Analytics client library. It provides asynchronous methods to access a specific use of Text Analytics, such as language detection or key phrase extraction.
-
-### Input
-
-A **document** represents a single unit of input to be analyzed by the predictive models in the Text Analytics service. Operations on `TextAnalyticsClient` take a collection of inputs to be analyzed as a batch. The operation methods have overloads that allow the inputs to be represented as strings, or as objects with attached metadata.
-
-For example, each document can be passed as a string in an array, e.g.
-
-```typescript
-const documents = [
-  "I hated the movie. It was so slow!",
-  "The movie made it into my top ten favorites.",
-  "What a great movie!"
-];
-```
-
-or, if you wish to pass in a per-item document `id` or `language`/`countryHint`, they can be given as a list of `TextDocumentInput` or `DetectLanguageInput` depending on the operation;
-
-```javascript
-const textDocumentInputs = [
-  { id: "1", language: "en", text: "I hated the movie. It was so slow!" },
-  { id: "2", language: "en", text: "The movie made it into my top ten favorites." },
-  { id: "3", language: "en", text: "What a great movie!" }
-];
-```
-
-See [service limiations][data_limits] for the input, including document length limits, maximum batch size, and supported text encodings.
-
-### Return Value
-
-The return value corresponding to a single document is either a successful result or an error object. Each `TextAnalyticsClient` method returns a heterogeneous array of results and errors that correspond to the inputs by index. A text input and its result will have the same index in the input and result collections. The collection may also optionally include information about the input batch and how it was processed in the `statistics` field.
-
-An **result**, such as `AnalyzeSentimentResult`, is the result of a Text Analytics operation, containing a prediction or predictions about a single text input. An operation's result type also may optionally include information about the input document and how it was processed.
-
-The **error** object, `TextAnalyticsErrorResult`, indicates that the service encountered an error while processing the document and contains information about the error.
-
```

Comment:
In our track 2 sdks we always have these user scenario examples in the readme. Do we want to keep these on the autogenerated client just like the other readmes? @deyaaeldeen  @ramya-rao-a

Reply:
https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/metricsadvisor/ai-metrics-advisor#examples

Reply:
@KarishmaGhiya I am not sure if I correctly understand your comment. Do you mean why the code generator can not auto generate such user scenarios?

Reply:
Yes. So first, do we want to keep that section of "Examples" and if yes, then can the code generator autogenerate it? If not, then how was that there previously? Was this a manual sdk and then became auto-generated?